code/game/objects/items/weapons/storage/lockbox.dm

```diff
@@ -111,6 +111,28 @@
 	new /obj/item/clothing/accessory/medal/silver/valor(src)
 	new /obj/item/clothing/accessory/medal/heart(src)
 
+/obj/item/storage/lockbox/medal/cc
+	name = "central command medal box"
+	desc = "A locked box used to store ALL medals you could ever need."
+	max_combined_w_class = 30
+	storage_slots = 15
+	req_access = list(ACCESS_CENT_COMMANDER)
+
+/obj/item/storage/lockbox/medal/cc/populate_contents()
+	new /obj/item/clothing/accessory/medal/gold/heroism(src)
+	..()
+	new /obj/item/clothing/accessory/medal/gold(src)
```

Comment:
Any reason we call parent in the middle of this all? If not, could you move it to either the bottom or the top of this proc so it looks a little neater?

Reply:
CC medal, captain medals, all other medals